src/mp_api/client.py

```diff
@@ -390,12 +401,293 @@ def get_entries(
 
         else:
             for doc in self.thermo.search_thermo_docs(
-                chemsys_formula=chemsys_formula, all_fields=False, fields=["entries"],
+                chemsys_formula=chemsys_formula,
+                all_fields=False,
+                fields=["entries"],
             ):
                 entries.extend(list(doc.entries.values()))
 
             return entries
 
+    def get_pourbaix_entries(
+        self,
+        chemsys: Union[str, List],
+        solid_compat="MaterialsProject2020Compatibility",
+        use_gibbs: bool = False,
```

Comment:
I like `use_gibbs`, two comments: (1) should this not be an int (i.e. the temperature)? and (2) shall we add this to `get_entries` for consistency too?

Reply:
1. I like this idea in principle. Rigorously though, `MaterialsProjectAqueousCompatibility` corrections are only valid for 298K, so using a different temperature here would be a bit inconsistent. If we _were_ to make this an `int` as you suggest, maybe we should just change the kwarg to `temperature` with a default value of 0 (in which case we don't use the Gibbs model)?
2. Sure, can do

Reply:
re: 1) I realized that I don't like the idea of making the kwarg `temperature` because when you activate the Gibbs model it changes the meaning of the energy, from DFT energy to free energy of formation. I feel like the None / int dichotomy does a better job signaling that to users than a continuous numerical argument would

Reply:
Re 2): the problem with integrating this into `get_entries` is that you need a complete phase diagram in order to use `GibbsComputedStructureEntry`. So if you try to use it on a typical `get_entries` query there is a substantial risk of hitting a `ValueError` when `GibbsComputedStructurEntry` tries to build a phase diagram

Reply:
Actually, easy solution - only allow this kwarg in `get_entries_in_chemsys`, not `get_entries`

Reply:
If it's only valid for 298K, that's fair, let's leave it as is.